test/pseudonymsys_test.go

```diff
@@ -83,3 +110,19 @@ func TestPseudonymsys(t *testing.T) {
 	assert.Nil(t, sessionKey2, "Authentication should fail, and session key should be nil")
 	assert.NotNil(t, err, "Should produce an error")
 }
+
+func getRegistrationKey() (string, error) {
```

Comment:
It's actually not about complexity at all. I am jut not too happy about having this here, because I think it's a bit out of scope, since emmy server is never aware of what kind of keys it checks ("real" or not), it only cares about their presence/absence or expiration. Besides, if we're externally generating registration keys in a similar fashion as in `getRegistrationKey()`, it does not mean that anyone else will use such form of registration keys. My point is, since registration keys are generated externally, they can really be *anything*. So I think we shouldn't bother with this here.

I still insist/propose that we have a function `insertTestRegistrationKeys()` that inserts some dummy keys. 

PS: I know that `checkRegistrationKey` deletes the key from the DB if everything goes well, but... just thinking, does it make sense to try to clean up after the tests (I'm thinking also `remove/delete/cleanupTestRegistrationKeys()`, whichever you prefer)? Maybe not, decide as you wish.